tutorial/source/bayesian_regression.ipynb

```diff
@@ -182,23 +190,25 @@
     "\n",
     "### `random_module()`\n",
     "\n",
-    "In order to do this, we'll 'lift' the parameters $w$ and $b$ to random variables. We can do this in Pyro via `random_module()`, which effectively takes a `nn.Module` and turns it into a distribution over neural networks. Specifically, each parameter in the original neural net is sampled from the provided prior. This allows us to repurpose vanilla neural nets for use in the Bayesian setting. For example:"
+    "In order to do this, we'll 'lift' the parameters $w$ and $b$ to random variables. We can do this in Pyro via `random_module()`, which effectively takes a `nn.Module` and turns it into a distribution over models. Specifically, each parameter in the original model is sampled from the provided prior. This allows us to repurpose vanilla regression models for use in the Bayesian setting. For example:"
```

Comment:
isnt this conflating a "model" for regression with the `model()` used for svi?  this reads to a beginner like they are the same thing when they are in fact different. or i guess it might be worth connecting `nn.Module` to a "regression model" (eg saying that were using a nn to represent this regression model)